config/config.yaml

```
@@ -0,0 +1,12 @@
+---
+db:
+  host: 192.168.178.20
+  port: 5432
+  user: postgres
+  password: postgres
+  name: mydatabase
+  providers_table_name: provider_service_table
+worker:
+  count: 5
+general:
+  development: true
```

Let's make sure we can run under different profiles, at least: 
development, testing, staging, production

```suggestion
runtimeEnvironment: 
  profile: "development"
```